utils/helper.py

```diff
@@ -24,3 +30,24 @@ def calculate_earning(task):
     rate = task.rate_per_hour
     total_earning = time * rate
     return total_earning
+
+
+def my_scheduled_task():
```

Comment:
create a file-like tasks.py or scheduler.py  inside the particular app dir itself. So, we can have multiple scheduler functions for each app separately and we can import and use them in our common cron scheduler.